circus-train-s3-mapreduce-cp-copier/pom.xml

```diff
@@ -37,6 +37,11 @@
       <artifactId>circus-train-aws</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
This copier should not depend on gcp. The discp-copier also shouldn't. If there is any class in gcp that is used it should go to some common module.

Reply:
Changed so that the circus-train-gcp and circus-train-aws are no longer required in copiers